concept_map.puml

```diff
@@ -53,12 +60,20 @@ CI -[hidden]down-> Hosting
 Hosting -[hidden]right-> Authentication
 Authentication -[hidden]right-> Firestore
 
-Firebase -[hidden]right-> Core
+Web --up--> Firestore : reads metrics and \l stores app config
+Web --up--> Authentication : authenticates
+Web --up--> Hosting : deployed on
+Web --up--> Analytics : reports analytics data
+Web -down-> Sentry : reports about errors
```

Comment:
```suggestion
Web -down-> Sentry : logging
```
Actually, Sentry may be used not for errors only so maybe we can write logging here.